### PR TITLE
Place item

### PR DESCRIPTION
**Pull request by faraidoun**

### CODE REVIEW

File: place-item.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+        <title>Home page</title>
+        <meta name="author" content="">
```
Comment (Mekaeil) [inline place-item.html]: Please edit title and meta tags, use correct names, use WeProDev name ,....

File: sass/modules/item/_place-item.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
+/*========================================================================================
+*********    
+*******      
+*****       PLACE ITEM STYLES
+***         
+*/
+
+.place-item{
```
Comment (Mekaeil) [inline sass/modules/item/_place-item.scss]: Please write cleaner codes. All codes must be in the same order!

**Comment by faraidoun** [comment]
pleas see changes

File: place-item.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    <meta name="description" content="Ready To Design">
+    <meta name="robots" content="index follow">
+    <meta name="copyright" content="2017-2018">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:url" content="http://WeProDev.com">
+    <meta name="twitter:domain" content="http://mekaeil.me">
+    <meta name="twitter:title" content="Menu Food">
+    <meta name="twitter:description" content="Ready To Design">
+    <meta name="twitter:image" content="http://mekaeil.me">
```
Comment (Mekaeil) [inline place-item.html]: Please edit meta tags, remove my name and use the Weprodev name.

**Comment by faraidoun** [comment]
i change meta tags

**Pull request merged**